### PR TITLE
Tests code coverage

### PR DESCRIPTION
**Pull request by programmer4web**
fixes https://github.com/Neovici/cosmoz-bottom-bar/issues/22

### CODE REVIEW

File: cosmoz-bottom-bar-view.js [hunk 1]
```diff
@@ -102,7 +102,7 @@
 			this._lastScroll = scrollTop;
 		},
 
-		_computeScrollerContentStyle: function (barHeight, bottomBarVisible, fixed) {
+		_computeScrollerContentStyle: function (barHeight, bottomBarVisible) {
```
Comment (nomego) [inline cosmoz-bottom-bar-view.js]: @plequang Did we drop the `fixed` stuff completely?

File: cosmoz-bottom-bar-view.js [hunk 2]
```diff
@@ -90,7 +90,7 @@
 			}
 		},
 
-		_scrollManagement: function (e) {
+		_scrollManagement: function () {
```
Comment (nomego) [inline cosmoz-bottom-bar-view.js]: `function` keyword not needed

Comment (programmer4web) [reply cosmoz-bottom-bar-view.js]: ok I will remove that

File: cosmoz-bottom-bar-view.js (hunk 1 above)
Comment (plequang) [reply cosmoz-bottom-bar-view.js]: `fixed` has been dropped from `cosmoz-bottom-bar`,  but it's still there in `cosmoz-bottom-bar-view`, when we don't want the bottom-bar to auto-hide when scrolling.

**Pull request merged**